js/console_form_handler.js

```diff
@@ -85,6 +83,7 @@ $(function()
                         me.element.find('.last-error').text(data.error);
                     }
                     me._onUpdateAlways(data, sFormPath);
+	                me.element.find('[data-field-id=previous_values]').find('input[type=hidden]').val('{}');
```

Comment:
Please put quotes around attribute value in selectors.

```suggestion
	                me.element.find('[data-field-id="previous_values"]').find('input[type="hidden"]').val('{}');
```